Makefile

```diff
@@ -81,13 +81,16 @@ dynamodb-create-table:
 dynamodb-scan-table:
 	aws dynamodb scan --table-name claudie --endpoint-url http://localhost:8000 --no-cli-pager
 
-# we need the value of local architecture to pass to docker as a build arg and
+# We need the value of local architecture to pass to docker as a build arg and
 # Go already needs to be installed so we make use of it here.
+# Use sed to set the image tag for cluster adapter, clean up at the end.
 TARGETARCH = $$(go env GOHOSTARCH)
 REV = $$(git rev-parse --short HEAD)
 SERVICES = $$(command ls services/)
 containerimgs:
+	sed -i "s/image: ghcr.io\/berops\/claudie\/autoscaler-adapter/&:$(REV)/" services/kuber/templates/autoscaler-adapter.goyaml	
 	for service in $(SERVICES) ; do \
-		echo " --- building $$service"; \
-		DOCKER_BUILDKIT=1 docker build --build-arg=TARGETARCH="$(TARGETARCH)" -t "ghcr.io/claudie/$$service:$(REV)" -f ./services/$$service/Dockerfile . ; \
-	done
\ No newline at end of file
+		echo " --- building $$service --- "; \
+		DOCKER_BUILDKIT=1 docker build --build-arg=TARGETARCH="$(TARGETARCH)" -t "ghcr.io/berops/claudie/$$service:$(REV)" -f ./services/$$service/Dockerfile . ; \
+	done
+	sed -i "/image: ghcr.io\/berops\/claudie\/autoscaler-adapter:/c\          image: ghcr.io\/berops\/claudie\/autoscaler-adapter" services/kuber/templates/autoscaler-adapter.goyaml
```

Comment:
This line is removing the already inserted tag, to make sure the template is left untouched after the build. While I do agree that the exact amount of whitespace is too fragile, I did not want to spend more than 5 min on this, as this make rule is rarely used. If you have a better one, please share :)

Reply:
I see. I might try to conjure some sed/awk magic for you.

also, why do we care if the template stays modified after the build?
because it would become part of the changes committed?

Reply:
>also, why do we care if the template stays modified after the build?
because it would become part of the changes committed?

Yes, it would pollute the template

Reply:
if that's the only change performed on the file - couldn't we just `git restore` it?

Reply:
Not exactly, as `git restore` would revert all changes in that file. We just need to remove a tag, added by the previous `sed` command. If there would be any changes to that manifest, they would be lost.

Reply:
in that case, the promised incantation:

```sh
# try out on a clean file with:
#   sed 's/adapter$/adapter:haha/' services/kuber/templates/autoscaler-adapter.goyaml | sed 's/adapter:.*$/adapter/'

sed -i 's/adapter:.*$/adapter/' services/kuber/templates/autoscaler-adapter.goyaml
```

Reply:
Looks like it works, thanks!